### ISSUE TITLE
Rearrange drop-down menu controls/buttons

### ISSUE THREAD

**Issue by ghost** [denschub/firefox-tabgroups#52]
Great add-on, been using it this whole year with no major problems.  One thing I noticed at first in the tab group menu was how close the 'drop down' (v) and 'close group' (x) buttons are to each other.  I thought it inevitable that someday I'd accidentally close a tab group that was meant to be dropped open, and I just did.  Restored it though, with the Session Manager add-on, but still...
I saw another issue asking about group closure confirmation (#9), and being able to undo or reopen a tab group would be a good feature, but maximizing the distance between the 'close group' button (x) and the 'drop down' button (v), placing the (v) on the far left, for example, would eliminate the risk of accidentally closing a tab group, and the need for closure confirmation or a reopen/undo feature as most users would only rethink or regret closing a tab group that was closed by mistake. (but it would also be nice having the option to reopen an entire tab group)
TL;DR, move the X away from the V, too close

**Comment by skyschub** [comment]
Accidental closing of groups will get addressed in #34, but we can leave this open to have further discussion once we're done with that PR.
